components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/impl/FederatedAuthenticatorConfigBuilderFactory.java

```diff
@@ -435,6 +466,21 @@ private static void resolveEndpointConfiguration(FederatedAuthenticator authenti
         }
     }
 
+    private static List<String> resolveAuthenticatorTags(FederatedAuthenticatorConfig config) {
+
+        /* If the authenticator is defined by the user, return the tags of the authenticator config. Otherwise, return
+        the tags of the system registered federated authenticator template.
+         */
+        if (DefinedByType.USER == config.getDefinedByType()) {
+            return Arrays.asList(config.getTags());
+        }
+
+        FederatedAuthenticatorConfig federatedAuthenticatorConfig =
+                ApplicationAuthenticatorService.getInstance().getFederatedAuthenticatorByName(config.getName());
+        return federatedAuthenticatorConfig != null ? Arrays.asList(federatedAuthenticatorConfig.getTags())
```

Comment:
Same as above aomment

Reply:
Revert changes done for https://github.com/wso2/identity-api-server/pull/758#discussion_r1881282158